modules/admission-controller-default.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * architecture/admission-controllers.adoc
+
+[id="admission-controller-default_{context}"]
+= Default admission controllers
+
+//Future xref - A set of default admission controllers are enabled in {product-title} {product-version}. These default admission controllers contribute to fundamental control plane functionality, such as ingress admission policy, xref:../nodes/clusters/nodes-cluster-overcommit.adoc#nodes-cluster-resource-override_nodes-cluster-overcommit[cluster resource limit override] and quota policy.
+A set of default admission controllers are enabled in {product-title} {product-version}. These default admission controllers contribute to fundamental control plane functionality, such as ingress admission policy, cluster resource limit override and quota policy.
+
+Admission plug-ins are resource specific. As such, resources served by the API server use a different admission chain.
```

Comment:
No. The chain is the same. But plugins ignore most resources they are not responsible for.

Reply:
Thank you for clarifying. This has now been reflected in my update to the doc. I have also moved this to the initial 'about' section and structured it to flow better.